src/test/java/com/hotels/mutantswarm/mutate/LexerMutatorStoreTest.java

```diff
@@ -40,6 +43,10 @@
   private Imposter imposter1, imposter2, imposter3;
   @Mock
   private List<CommonToken> stream;
+  @Mock
+  private Imposters.Builder impostersBuilder;
+  @Mock
+  private Map<Integer, Imposter> imposters = new HashMap<>();
```

Comment:
Is this map actually being used anywhere?

Reply:
Ditto for the `impostersBuilder`